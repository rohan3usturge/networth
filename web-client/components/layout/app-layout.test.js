import withAppLayout from "./app-layout";
import { shallow } from "enzyme";

describe("AppLayout", () => {
  it("should render properly and match snapshot", () => {
    const Dummy = withAppLayout(() => <span>Hello</span>);
    const wrapper = shallow(<Dummy />);
    expect(wrapper).toMatchSnapshot();
  });
});
