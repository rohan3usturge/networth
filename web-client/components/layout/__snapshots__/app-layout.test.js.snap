// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppLayout should render properly and match snapshot 1`] = `
<section
  className="bg-light"
>
  <Header />
  <div
    className="mt-5"
  />
  <main
    className="container-md pl-1 pr-1"
  >
    <div>
      <Component />
    </div>
  </main>
  <Footer />
</section>
`;
